examples/helpers/mail/mail_example.py

```diff
@@ -104,22 +108,28 @@ def build_kitchen_sink():
     mail_settings = MailSettings()
     mail_settings.bcc_settings = BCCSettings(True, Email("test@example.com"))
     mail_settings.bypass_list_management = BypassListManagement(True)
-    mail_settings.footer_settings = FooterSettings(True, "Footer Text", "<html><body>Footer Text</body></html>")
+    mail_settings.footer_settings = FooterSettings(
```

Comment:
can you please move the params to each be on their own line?

Reply:
Updated.